templates/signup.html

```diff
@@ -133,17 +170,23 @@ <h4 class="text-blue-500 mt-0 mb-2 text-center">{{_('create_account_explanation'
                 </div>
             </div>
             <div class="flex flex-row items-center justify-center px-4 md:px-8 gap-4">
+
+            {% if is_teacher %}
                 <input type="checkbox" id="agree_third_party" name="agree_third_party">
                 <label class="w-96 text-center" for="agree_third_party">{{_('agree_third_party')}}</label>
```

Comment:
Why is this only allowed when creating a teacher account?

Reply:
This is a change I made: we only send emails from third parties to teachers (this is already current practice) we are never going to try to market stuff to kids :)